@navikt/core/react/src/overlay/dismiss/DismissableLayer.tsx

```diff
@@ -351,10 +375,10 @@ const DismissableLayerNode = forwardRef<HTMLDivElement, DismissableLayerProps>(
         onBlurCapture={focusOutside.onBlurCapture}
         onPointerDownCapture={pointerDownOutside.onPointerDownCapture}
         style={{
-          pointerEvents: pointerEnabled.isBodyPointerEventsDisabled
-            ? pointerEnabled.isPointerEventsEnabled
+          pointerEvents: isBodyPointerEventsDisabled
+            ? isPointerEventsEnabled
               ? "auto"
-              : "none"
+              : undefined
             : undefined,
```

Comment:
This can be simplified now:
```js
            isBodyPointerEventsDisabled && isPointerEventsEnabled
              ? "auto"
              : undefined,
```